File: formacao_cientista/4. estatistica_III/poisson.py
# -*- coding: utf-8 -*-
from scipy.stats import poisson

# media de acidentes = lambda = 2 / dia
# prob(x = 3) = ?

lambd = 2
print(poisson.pmf(3, lambd))

# media de acidentes = lambda = 2 / dia
# prob(x <= 3) = ?

lambd = 2
print(poisson.cdf(3, lambd))

# media de acidentes = lambda = 2 / dia
# prob(x > 3) = ?

lambd = 2
print(poisson.sf(3, lambd))

File: formacao-cientista/4. estatistica_III/poisson.R
# Poisson
# prob(x=a)
# dpois(x, lambda)

# media de acidentes = lambda = 2 / dia
# prob(x = 3) = ?

lambda = 2
dpois(3, lambda)

# prob(x<a)
# ppois(x, lambda)

# media de acidentes = lambda = 2 / dia
# prob(x <= 3) = ?

ppois(3, lambda)

# media de acidentes = lambda = 2 / dia
# prob(x > 3) = ?

ppois(3, lambda, lower.tail=F)

